.idea/misc.xml

```diff
@@ -8,7 +13,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="11" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
changes like this one :P

Reply:
Yeah, this was mainly from me using two separate JDKs, and starting out with JDK 11... This commit is just me putting egg on my face.